src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -2062,7 +2062,7 @@ def test_ingress_config_multi_vips(self, _run_cephadm, cephadm_module: CephadmOr
                                 'http-request use-service prometheus-exporter if { path /metrics }\n    '
                                 'monitor-uri /health\n'
                                 '\nfrontend frontend\n    '
-                                'bind *:8089\n    '
+                                'bind [..]:8089\n    '
```

Comment:
A few lines above this, there is another `bind *:8999\n` line that needs the same adjustment. Currently the unit test fails
```
E                 Full diff:
E                   {
E                    'files': {'haproxy.cfg': '# This file is generated by cephadm.\n'
E                                             'global\n'
E                                             '    log         127.0.0.1 local2\n'
E                                             '    chroot      /var/lib/haproxy\n'
E                                             '    pidfile     /var/lib/haproxy/haproxy.pid\n'
E                                             '    maxconn     8000\n'
E                                             '    daemon\n'
E                                             '    stats socket /var/lib/haproxy/stats\n'
E                                             '\n'
E                                             'defaults\n'
E                                             '    mode                    http\n'
E                                             '    log                     global\n'
E                                             '    option                  httplog\n'
E                                             '    option                  dontlognull\n'
E                                             '    option http-server-close\n'
E                                             '    option forwardfor       except 127.0.0.0/8\n'
E                                             '    option                  redispatch\n'
E                                             '    retries                 3\n'
E                                             '    timeout queue           20s\n'
E                                             '    timeout connect         5s\n'
E                                             '    timeout http-request    1s\n'
E                                             '    timeout http-keep-alive 5s\n'
E                                             '    timeout client          30s\n'
E                                             '    timeout server          30s\n'
E                                             '    timeout check           5s\n'
E                                             '    maxconn                 8000\n'
E                                             '\n'
E                                             'frontend stats\n'
E                                             '    mode http\n'
E                 -                           '    bind *:8999\n'
E                 ?                                     ^
E                 +                           '    bind [..]:8999\n'
E                 ?                                     ^^^^
E                                             '    bind 1.2.3.7:8999\n'
E                                             '    stats enable\n'
E                                             '    stats uri /stats\n'
E                                             '    stats refresh 10s\n'
E                                             '    stats auth admin:12345\n'
E                                             '    http-request use-service prometheus-exporter if '
E                                             '{ path /metrics }\n'
E                                             '    monitor-uri /health\n'
E                                             '\n'
E                                             'frontend frontend\n'
E                                             '    bind [..]:8089\n'
E                                             '    default_backend backend\n'
E                                             '\n'
E                                             'backend backend\n'
E                                             '    option forwardfor\n'
E                                             '    balance static-rr\n'
E                                             '    option httpchk HEAD / HTTP/1.0\n'
E                                             '    server rgw.foo.test.gwznjd 1.2.3.7:80 check '
E                                             'weight 100\n'},
E                   }

```

Reply:
Oops. Allright, should be fixed.
Let's see if the test pass this time. Thanks for the comment.